src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -374,6 +374,7 @@ - (BOOL)startup:(MTRDeviceControllerStartupParamsInternal *)startupParams
         // bring-up.
         commissionerParams.removeFromFabricTableOnShutdown = false;
         commissionerParams.deviceAttestationVerifier = _factory.deviceAttestationVerifier;
+        commissionerParams.permitMultiControllerFabrics = startupParams.isAdditionalController;
```

Comment:
minor nit: does permitMultiControllerOnFabric make more sense here :)

Reply:
> does permitMultiControllerOnFabric make more sense here

It depends on how you think about it, but either way out of scope for this PR: permitMultiControllerFabrics already exists in the core SDK and we are just using it here.